config/default/manager_metrics_patch.yaml

```diff
@@ -1,4 +1,7 @@
 # This patch adds the args to allow exposing the metrics endpoint securely
 - op: add
-  path: /spec/template/spec/containers/0/args/0
+  path: /spec/template/spec/containers/0/args/-
   value: --metrics-bind-address=:8080
+- op: add
```

Comment:
installation will fail, if there is no mounted secrets. 

Also, it's a bit weird, that manager_metrics_patch configures webook for admission controllers.

Reply:
I'd suggest it either move to webhook related configs somehow or keep it as part of manifests CSV generation commands

Reply:
updated manifests and tested locally on kind with cert-manager installed